packages/date-time-picker/test/value-commit.test.js

```diff
@@ -0,0 +1,430 @@
+      await sendMouseToElement({ type: 'click', element: otherPicker });
+      await expectValueCommit();
+    });
+
+    it('should commit on date-time-picker outside click', async () => {
+      outsideClick();
+      await expectValueCommit();
+    });
+  });
+
+  describe('initially valid value', () => {
+    beforeEach(async () => {
+      dateTimePicker.value = '2002-02-02T12:00';
+      await nextRender();
+      resetSpyHistories();
+    });
+
+    describeForEachPicker('value partially cleared (= incomplete value)', (pickerType) => {
+      let picker, otherPicker;
+
+      beforeEach(async () => {
+        [picker, otherPicker] = getPicker(pickerType);
+        await clearInput(picker);
+      });
+
+      it('should not commit on Enter', async () => {
+        await sendKeys({ press: 'Enter' });
+        await expectNoValueCommit();
+      });
+
+      it('should not commit on other picker click', async () => {
+        await clearInput(picker);
+        await sendMouseToElement({ type: 'click', element: otherPicker });
+        await expectNoValueCommit();
+      });
+
+      it('should commit on date-time-picker outside click', async () => {
+        await clearInput(picker);
+        outsideClick();
+        await expectValueCommit();
+      });
+    });
+  });
+  describe('initially invalid value', () => {
+    beforeEach(async () => {
+      dateTimePicker.min = '2011-01-01T12:00';
+      await enterInput(datePicker, '2/2/2002');
+      await sendKeys({ press: 'Enter' });
+      await enterParsableInput(timePicker);
+      outsideClick();
+      await nextRender();
+      resetSpyHistories();
+    });
+
+    describeForEachPicker('value partially cleared (= incomplete value)', (pickerType) => {
+      let picker, otherPicker;
+
+      beforeEach(() => {
+        [picker, otherPicker] = getPicker(pickerType);
+      });
+
+      it('should commit on Enter', async () => {
+        await clearInput(picker);
+        await sendKeys({ press: 'Enter' });
+        await expectValueCommit();
+      });
+
+      it('should commit on other picker click', async () => {
+        await clearInput(picker);
+        await sendMouseToElement({ type: 'click', element: otherPicker });
+        await expectValueCommit();
+      });
+
+      it('should commit on date-time-picker outside click', async () => {
+        await clearInput(picker);
+        outsideClick();
+        await expectValueCommit();
+      });
+    });
+  });
+
+  describe('initially invalid empty field', () => {
+    beforeEach(() => {
+      dateTimePicker.required = true;
+      dateTimePicker.validate();
+    });
+
+    describeForEachPicker('incomplete input entered', (pickerType) => {
+      let picker, otherPicker;
+
+      beforeEach(() => {
+        [picker, otherPicker] = getPicker(pickerType);
+      });
+
+      it('should commit as unparsable on Enter', async () => {
+        await enterParsableInput(picker);
```

Comment:
Let's move this to `beforeEach`

```suggestion
        await enterParsableInput(picker);
```

Reply:
Moved.